### ISSUE TITLE
Copy email address onto clipboard

### ISSUE THREAD

**Issue by PaulKuykendall** [cisagov/manage.get.gov#1848]
### Issue description

Problem: analyst need to copy an email address cleanly so that they can perform searches to verify the contact information through public sources. Sometimes, it's difficult to cleanly highlight and copy this email address, and it was a certain amount of time to get it correct.

Solution:Analysts want to have a BUTTON to copy an email address cleanly onto the clipboard so that it may be used in searches to verify the contact information through public sources.

### Acceptance criteria

- [ ] a button exists by each email address, which will copy the email address cleanly into the clipboard

### Additional context

See diagrams

**Comment by vickyszuchin** [comment]
Block tag added: this ticket (currently planned for sprint 41) is blocked by another ticket (#1852) in the same sprint.

**Comment by abroddrick** [comment]
Added the requirement that this shows only for analysts
